src/UI/Implementation/Component/Input/Field/Radio.php

```diff
@@ -132,19 +132,23 @@ public function withInput(PostData $post_input) {
 
 			foreach ($dep_fields as $name => $field) {
 				$filled = $field->withInput($post_input);
-
 				$content = $filled->getContent();
+
 				if ($content->isOk()) {
 					$values['group_values'][$name] = $content->value();
 				} else {
-					$error = true;
+					$clone = $clone->withError($name . $content->error());
```

Comment:
We seem to be missing some delimiter between the name and the error (e.g. ": ") here, if I understand correctly.

Reply:
you are right; I added a "-".

Reply:
Give it some space " - " =) . Thx!